### PR TITLE
[mtl-2019] schedule update1, add sponsor

### PR DESCRIPTION
**Pull request by pdion891**
- Update schedule
- add sponsor
- minor adjustment in welcome page.

### CODE REVIEW

**Comment by purpleidea** [comment]
Wait. Do not merge yet. Why did you make these unrelated schedule changes??

**Comment by MikeRosTX** [comment]
When you're ready for it to me merged, let us know.

File: data/events/2019-montreal.yml [hunk 1]
```diff
@@ -355,48 +360,42 @@ program:
...
-  - title: "martin-deslongchamps"
-    type: talk
-    date: 2019-11-06
-    start_time: "14:30"
-    end_time: "14:55"
-
   - title: "Open Space Opening"
     type: open-space
     date: 2019-11-06
-    start_time: "14:55"
```
Comment (purpleidea) [inline data/events/2019-montreal.yml]: What's this??

**Comment by bridgetkromhout** [comment]
@pdion891 Perhaps it would be best if you removed the schedule changes from this PR and only submitted the sponsor addition. We could then merge that and get your sponsor listed while your team discusses the schedule.

**Comment by purpleidea** [comment]
@bridgetkromhout It shouldn't be too long, pdion and I just finished discussing, and we have a new version that he should be sending as an updated PR shortly. Thank you!

File: data/events/2019-montreal.yml (hunk 1 above)
Comment (pdion891) [reply data/events/2019-montreal.yml]: reviewed via phone

**Comment by purpleidea** [comment]
@pdion891 Can you squash this into a single commit so it's easier to see the yaml changes please?

**Comment by pdion891** [comment]
Hi @bridgetkromhout  I wan't expected too much delay for the scheduled part if we do have some than I'll split merge, but shouldn't have to.

@purpleidea changes updated has discussed.

**Comment by purpleidea** [comment]
I think it looks good. We can merge @MikeRosTX

**Comment by pdion891** [comment]
thanks for the merge !

**Pull request merged**